contracts/pricers/STETHPricer.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+import {OracleInterface} from "../interfaces/OracleInterface.sol";
+import {OpynPricerInterface} from "../interfaces/OpynPricerInterface.sol";
+import {STETHInterface} from "../interfaces/STETHInterface.sol";
+import {ERC20Interface} from "../interfaces/ERC20Interface.sol";
+import {SafeMath} from "../packages/oz/SafeMath.sol";
+
+/**
+ * @notice A Pricer contract for a Yearn yToken
+ */
+contract STETHPricer is OpynPricerInterface {
```

Comment:
I think we should probably call it WSTETHPricer as it is pricing wstETH collateral.